### ISSUE TITLE
Desktop icons not aligned in Debian 9

### ISSUE THREAD

**Issue by mYnDstrEAm** [linuxmint/nemo#1552]
```
 * nemo 3.2.2
 * Debian 9.0 with Cinnamon
 * nvidia-driver
 * 64 bit
 * 2 monitors
 ```

**Issue**

Desktop icons are not aligned to some sort of grid even though "Keep aligned" is ticked (rechecking it doesn't help either)  
They can be moved by small margins.  
Sometimes they overlap.

**Expected behaviour**

They should be aligned by an invisible grid and snap to their position in its tiles and stay where they are.

**Comment by leigh123linux** [comment]
Closed as 'wontfix', update to nemo-3.4.x

**Comment by mYnDstrEAm** [comment]
@leigh123linux Um how?! Synaptic tells me that this would be the latest version.

**Comment by leigh123linux** [comment]
@mYnDstrEAm If debian chooses to ship an old version that is now EOL it is their job to fix it.

**Comment by mYnDstrEAm** [comment]
Okay. Where can people best request this to be done? Or can you ask them? Maybe they do have good reasons...
@marga-debian @maxyz

**Comment by maxyz** [comment]
@leigh123linux we (Debian hat on) still need to support 3.2.x for a couple of more years, I consider this issue not important enough to backport a fix to 3.2, and I agree with the 'wontfix' as a wont backport.
@mYnDstrEAm it will be updated in sid/testing when we have the time to work on cinnamon.
Please report the Debian specific issues to the Debian bugtracker: https://bugs.debian.org using the reportbug or reportbug-ng tool.

**Comment by leigh123linux** [comment]
@maxyz nemo-desktop grid feature fixes the issue but it is too complicated to backport.
